src/components/navbar.component.tsx

```diff
@@ -15,7 +26,7 @@ export interface NavbarProps {
 
 const Navbar: React.FC<NavbarProps> = ({ user, onLogout }) => {
   const [sidenavOpen] = React.useState(false);
-  const [isUserMenuOpen, setIsUserMenuOpen] = React.useState(false);
+  const [isUserMenuOpen, setIsUserMenuOpen] = React.useState<boolean>(false);
```

Comment:
The `boolean` is not needed - it is already a boolean due to `false`. Type inference works great - you can leverage it when possible.